tests/acceptance/features/bootstrap/Provisioning.php

```diff
@@ -4294,8 +4295,12 @@ public function theDisplayNameReturnedByTheApiShouldBe(string $expectedDisplayNa
 	 */
 	public function theDisplayNameOfUserShouldBe(string $user, string $displayname):void {
 		$actualUser = $this->getActualUsername($user);
-		$this->retrieveUserInformationAsAdminUsingProvisioningApi($actualUser);
-		$this->theDisplayNameReturnedByTheApiShouldBe($displayname);
+		$response = $this->retrieveUserInformationAsAdminUsingProvisioningApi($actualUser);
+		$responseDisplayName = (string) $this->getResponseXml($response, __METHOD__)->data[0]->displayname;
+		Assert::assertEquals(
+			$displayname,
+			$responseDisplayName
+		);
```

Comment:
there is this method we can use
```suggestion
		$this->theDisplayNameReturnedByTheApiShouldBe($displayname, $response);
```